com.chartboost.mediation/Runtime/Banner/ChartboostMediationBannerAdSize.cs

```diff
@@ -14,10 +12,10 @@ public enum ChartboostMediationBannerType
     
     public enum ChartboostMediationBannerSizeType
     {
-        Adaptive,
-        Standard,
-        Medium,
-        Leaderboard
+        Adaptive = -1,
```

Comment:
-1?? 🤔 What do our other platforms have for this?

Reply:
Other platforms don't have it. It is just a helper enum to differentiate between different size types. It's mainly used between Unity and native bridge

Reply:
Perhaps we can go with a different number and we can have `Unknown` as a -1? 🤔

Reply:
That's not a bad idea. I can make Adaptive as 4 and Unknown as -1